examples/http-client-oauth2-password-grant-type/http_client_oauth2_password_grant_type.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - OAuth2 password grant type
 
-A client, which is secured with OAuth2 password grant type can be used to connect to a secured service. The client is enriched with the `Authorization: Bearer <token>` header by passing the `http:OAuth2PasswordGrantConfig` to the `auth` configuration of the client.
+An `http:Client` can be secured with OAuth2 password grant type authorization. This is achieved by enriching each request from the client with the `Authorization: Bearer <token>` header. The required configurations for the OAuth2 password grant type can be specified in the `auth` field as the `http:OAuth2PasswordGrantConfig` record. Use this to communicate with the service, which is secured with OAuth2 authorization.
```

Comment:
**When to use ?**
use when you need to exchange a user's credentials for an access token. Use this only when there is high level of trust with the client. This grant type is not recommended as this grant type is deprecated per OAuth 2.1.

Reply:
Added via [187d99d](https://github.com/ballerina-platform/ballerina-distribution/pull/3793/commits/187d99dffcd52f92e019e05f224ac4dc5e4a2e66)

Reply:
I did not add the deprecated statement to the content. If this is deprecated we may remove this BBE right?